packages/gestalt/src/SideNavigation/NavigationContent.js

```diff
@@ -88,62 +91,65 @@ export default function NavigationContent({
   const normalWidth = 280;
   const headerWidth = isCollapsed ? 44 : undefined;
   const collapsedWidth = shouldCollapseEmpty ? 40 : 60;
-
-  const wrapperWidth = sideNavigationCollapsed ? collapsedWidth : normalWidth;
   const contentWidth = isCollapsed ? collapsedWidth : normalWidth;
 
   return (
     <Box
+      ref={mainContainer}
       aria-label={accessibilityLabel}
       as="nav"
       color="default"
       height="100%"
       minWidth={collapsible ? undefined : normalWidth}
       position="relative"
-      width={collapsible ? wrapperWidth : undefined}
+      width={sideNavigationCollapsed ? collapsedContainerWidth : undefined}
       zIndex={overlayPreview ? new FixedZIndex(1) : undefined}
     >
       <div
         ref={scrollContainer}
-        className={classnames(styles.fullHeight, layoutStyles.borderBox, {
+        className={classnames(styles.fullHeight, layoutStyles.overflowAutoY, {
```

Comment:
the main purpose of this 2nd wrapper is now to render the scrollbar. it is responsible for the scrolling of the sidenavigation.

prevoiusly is was also responsible for preserving the width and animating it. But that responnsiblity is now on the 3rd inner wrapper below.